src/Makefile

```diff
@@ -772,6 +782,7 @@ ifneq (${uname_S},HP-UX)
 	${MAKE} ${BUILD_CMAKE_PROJECTS}
 endif
 	${MAKE} ${BUILD_AGENT}
+	@${MAKE} strip DISABLE_STRIP_SYMBOLS=${DISABLE_STRIP_SYMBOLS}
```

Comment:
Ditto.
Also keep in mind that macos TARGET also will execute this. @RamiroRD eyes here